src/rules/enforce-boolean-naming-prefixes.ts

```
@@ -327,9 +388,17 @@ export const enforceBooleanNamingPrefixes = createRule<Options, MessageIds>({
           node.init.callee.type === AST_NODE_TYPES.Identifier
         ) {
           const calleeName = node.init.callee.name;
+          const lowerCallee = calleeName.toLowerCase();
+
+          // For assert*-style utilities, only treat as boolean if we can confirm boolean return type
+          if (lowerCallee.startsWith('assert')) {
+            return identifierReturnsBoolean(calleeName);
+          }
+
           // Check if the function name suggests it returns a boolean
           return approvedPrefixes.some((prefix) =>
```

ESLint

[error] Insert `⏎············`

(prettier/prettier)